src/app/app-routing.module.ts

```diff
@@ -13,6 +13,14 @@ export const routes: Routes = [
     pathMatch: 'full'
   },
   {
+    path: 'home',
```

Comment:
Hmm, I seem to have not actually comitted all the changes to app-routing - as routes for public, signin and others are missing. I will have to check in the morning, as I've not got the laptop I was using on me right now.

To answer your question - home is no longer the "first route" as you are calling it, public is, as it's the landing page for the app right now - this allows us to present a logged in and not logged in experience - but as those changes are missing...